subsys/nrf_security/Kconfig.psa

```diff
@@ -74,6 +74,15 @@ config MBEDTLS_USE_PSA_CRYPTO
 	help
 	  Corresponds to MBEDTLS_USE_PSA_CRYPTO setting in mbed TLS config file.
 
+config MBEDTLS_PSA_KEY_SLOT_COUNT
+	int "Number of PSA key slots available"
+	default 32
```

Comment:
Maybe we should give this a range, to disallow 0 as in input.

Reply:
I added 1-32 since 32 is the default in MBedTLS.